etherpad/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM etherpad/etherpad
+
+ADD ./rootfs/defaults/settings.json /opt/etherpad-lite/settings.json
+
+EXPOSE 9001
```

Comment:
Can we make this 80?

Reply:
That would be a bad idea.

From [OpenShift docs](https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html):

> Because the container user is always a member of the root group, the container user can read and write these files. The root group does not have any special permissions (unlike the root user) so there are no security concerns with this arrangement. In addition, the processes running in the container must not listen on privileged ports (ports below 1024), since they are not running as a privileged user.

Reply:
Ah, thanks for the insight! Let's keep the default one then.